invenio_workflows/models.py

```diff
@@ -52,50 +63,115 @@ class ObjectVersion(object):
     RUNNING = 3
     WAITING = 4
     ERROR = 5
-    MAPPING = {"New": 0, "Done": 1, "Need action": 2,
-               "In process": 3, "Waiting": 4, "Error": 5}
 
-    @classmethod
-    def name_from_version(cls, version):
-        try:
-            return cls.MAPPING.keys()[cls.MAPPING.values().index(version)]
-        except ValueError:
+    @staticproperty
+    def labels():  # pylint: disable=no-method-argument
+        return {
+            0: "New",
```

Comment:
This inherits from `enum.Enum` and so I couldn't figure out a reasonable way of referencing INITIAL from within a property. The goal here is to be able to use `ObjectStatus.INITIAL.label`.